mobile/lib/modules/asset_viewer/views/gallery_viewer.dart

```diff
@@ -31,7 +31,7 @@ import 'package:openapi/api.dart' as api;
 
 // ignore: must_be_immutable
 class GalleryViewerPage extends HookConsumerWidget {
-  late List<Asset> assetList;
```

Comment:
Why changing late to `final` would fix the issue?

Reply:
This has no effect here. It doesn't fix the issue, it just should not have been assigned `late` as there is no case it is allowed to be `null`. Unless I'm massively misunderstanding something here...

`late` tells the compiler that while this variable is not guarenteed to be non-null at initialization, it will be initialized later (hence, `late`) to something non-null. It's dangerous and usually unnecessary. In this case, this is just `final`. The fix was removing the asset from the list lower down.